src/python/zquantum/qcbm/ansatz_utils.py

```diff
@@ -62,23 +49,13 @@ def get_entangling_layer_line_topology(
     """Builds a circuit representing an entangling layer according to the line topology.
 
     Args:
-        params (numpy.array): parameters of the circuit.
-        n_qubits (int): number of qubits in the circuit.
-        entangling_gate (str): gate specification for the entangling layer.
-
-    Returns:
-        Circuit: a zquantum.core.circuit.Circuit object
+        params: parameters of the circuit.
+        n_qubits: number of qubits in the circuit.
+        entangling_gate: gate specification for the entangling layer.
```

Comment:
The `entangling_gate` parameter is incorrectly typed as str instead of the `GatePrototype`.

Reply:
Fixed in https://github.com/zapatacomputing/z-quantum-qcbm/pull/40/commits/cccdac07f49cfd7e56fbafb95518a2b007cb0f45